src/components/SelectBox/SelectBox.tsx

```diff
@@ -6,27 +6,20 @@ interface SelectBoxProps {
   onOptionChange(event: React.ChangeEvent<HTMLSelectElement>): void;
 }
 
-class SelectBox extends React.Component<SelectBoxProps> {
-  constructor(props: SelectBoxProps) {
-    super(props);
-    this.onChange = this.onChange.bind(this);
-  }
+function SelectBox({ onOptionChange, filter }: SelectBoxProps) {
+  const onChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    if (onOptionChange) onOptionChange(event);
```

Comment:
```suggestion
    onOptionChange?.(event);
```

그나저나 onOptionChange가 없는 경우도 있나도? interface 상에서 optional은 아닌 것 같아서요!

Reply:
해당부분에 값이 없는 경우가 없어서 조건문이 필요가 없었네요..피드백 반영하였습니다!